<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My File Info</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.xpm</normaloff>icon.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="FileList">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>0</y>
      <width>551</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FileCount">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>File Count: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TotalSize">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="actionSave_List"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionSave_List">
   <property name="text">
    <string>Save List</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
